### ISSUE TITLE
Add an introductory lightbox

### ISSUE THREAD

**Issue by bsteinbeigle** [Methodician/guytime#118]

**Comment by bsteinbeigle** [comment]
@kaxline What would you think about adding an introductory lightbox after someone creates a new account and before he is defaulted to browse profiles. I'm thinking the lightbox would say something like 'Cheers for joining! Please keep in mind this app is not done yet so it lacks some of the bells and whistles you might not normally see. Thank you for joining.' I'll think of different phrasing but hopefully the above gives you some idea. Let me know what you think!

**Comment by kaxline** [comment]
Yeah, that sounds good. Want to mock it up and finalize the language?

**Comment by bsteinbeigle** [comment]
@kaxline Below is a mock-up. Basically, after someone inputs their email/phone number & password when registering a new account they would be directed to the following pop-up window before being able to input profile information. Inside the pop up the message should read..

 "Welcome fellas. This app was created to hopefully help make it easier for guys to find and develop platonic friendships near where they live or where their traveling to. We are in the very early stages of development, so please be patient with us if you come across a glitch or two. That said, we still have a lot of work to do including adding other functionality that make this service pretty sweet compared to other friend apps. Feel free to send me a message and let me know what you think. Also let me know if you come across a bug, troll, etc."  

Cheers, Brandon Steinbeigle"


![App Intro Message](https://user-images.githubusercontent.com/89544670/177433343-e004dee3-bfd8-44b4-a958-ed70ccd341c8.jpg)
